src/core/packages/apps/server-internal/src/bundle_routes/file_hash_cache.ts

```diff
@@ -20,10 +20,10 @@ export interface IFileHashCache {
 
 /** @internal */
 export class FileHashCache implements IFileHashCache {
-  private lru: LruCache<string, Promise<string>>;
+  private lru: LRUCache<string, Promise<string>>;
 
   constructor(maxSize: number = 250) {
-    this.lru = new LruCache(maxSize);
+    this.lru = new LRUCache({ max: maxSize });
```

Comment:
NIT perhaps we can rename the maxSize parameter of the constructor, as it seems to refer to the number of elements rather than the storage space occupied.

Reply:
Addressed in [`e4b963b` (#206225)](https://github.com/elastic/kibana/pull/206225/commits/e4b963be09c50e5f7d072f700e64ec793db65db7)